### PR TITLE
Buildfixes

### PR DESCRIPTION
**Pull request by gkiagia**
A few fixes for some build errors that I had on my linux machine.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -111,7 +111,7 @@ build(){
             export V4L2_CFLAGS="$(/usr/bin/pkg-config --cflags libv4l2) "
             export V4L2_LIBS="$(/usr/bin/pkg-config --libs libv4l2) "
             export PLATFORM_GLIB_LIBS="-lpthread -lffi -lrt -ldl -lresolv"
-            export PLATFORM_GSTREAMER_LIBS="-lgstvideoconvert -lgstpulse -lgstvideo4linux2 -lX11 -lXv -lgstallocators-1.0 -lGLU -lGL"
+            export PLATFORM_GSTREAMER_LIBS="-lgstvideoconvert -lgstpulse -lgstvideo4linux2 -lX11 -lXv -lgstallocators-1.0 -lGLU -lGL -lEGL"
```
Comment (sdroege) [inline build.sh]: This should rather be fixed by disabling EGL in the gst-plugins-bad build. We're going to use GLX in glimagesink anyway

**Comment by sdroege** [comment]
Everything but the -lEGL thing are merged now.

**Comment by gkiagia** [comment]
Thanks. Since -lEGL is not correct to be merged, I'll close this request now.

**Comment by sdroege** [comment]
Do you want to provide a fix for the build scripts to --disable-egl?

**Pull request closed without merging**